clients/privacy-center/pages/api/fides-js.ts

```diff
@@ -148,6 +209,9 @@ export default async function handler(
   if (!fidesJS || fidesJS === "") {
     throw new Error("Unable to load latest fides.js script from server!");
   }
+
+  const customFidesCss = await fetchCustomFidesCss(req);
```

Comment:
If this fails at runtime when we launch this because of a version mismatch or something similar, what do we do? That'd be a lot of production issues!

I'd recommend we let this fail open- if _anything_ goes wrong fetching the custom CSS, log an error and continue rendering the generic script result as-is today 👍

Reply:
I agree, and that's the current behavior. All the error handling happens inside `fetchCustomFidesCss` we'll either get the custom CSS string or null, both of which are handled appropriately when we inject that a few lines down.

Reply:
Hmm, I *almost* trust you, but I'm going to look a second time now... 🤔

Reply:
OK I see the `try`/`catch`, you've convinced me 👍